src-web/components/kappnav/ComponentView.jsx

```diff
@@ -283,6 +286,24 @@ class ComponentView extends Component {
     this.filterTable(e.target.value, 1, this.state.pageSize, this.state.totalRows)
   }
 
+  loadingReduxStore(result, applicationName) {
+    var components = []
+    var componentViewReduxData = {
+      "applicationName": applicationName,
+      "applicationNamespace": this.props.baseInfo.selectedNamespace
```

Comment:
Are you sure selectedNamespace is going to be the application namespace?  What happens if all namespaces are selected? Why not pass in the applicationNamespace to be sure?